DEVELOPMENT.md

```diff
@@ -420,3 +423,52 @@ The backend extension is discovered by adding the label and the annotations in t
 1. Multiple context roots can be specified by using the . character to separate them: e.g. if tekton-dashboard-endpoints: foo.bar then /v1/extensions/extension-name/foo is routed to /foo and /v1/extensions/extension-name/bar is routed to /bar at the extension.
 1. "/" is a reserved context root for javascript.  "tekton-dashboard-endpoints" must be always set if the extension has javascript.
 
+__Kubernetes Resources__
+
+These extensions give the dashboard the ability to show a list of resources for a given type. The type can be selected 
```

Comment:
Some suggestions to clarify that we're exposing kubernetes resource by `kind` (type is a little ambiguous here), and avoid coupling the docs too tightly to the current implementation of the raw resource view.

> These extensions give the dashboard the ability to show a list of resources for a given Kubernetes resource kind. A link to this list is exposed in the nav similar to bundled extensions. Selecting an item from the list provides a more detailed view of the selected resource.